modules/seqexec/engine/src/main/scala/seqexec/engine/Engine.scala

```diff
@@ -64,7 +65,7 @@ class Engine[D, U](stateL: Engine.State[D]) {
   def reload(seq: Sequence.State[IO], steps: List[Step[IO]]): Sequence.State[IO] =
     Sequence.State.reload(steps, seq)
 
-  def startSingle(c: ActionCoords): HandleType[EventResult.Outcome] = get.flatMap { st =>
+  def startSingle(c: ActionCoords, sys: Resource): HandleType[EventResult.Outcome] = get.flatMap { st =>
```

Comment:
isn't there a `Resource` in `ActionCoords`? If not that looks the right place to have it. @jluhrs WDYT?

Reply:
I don't think so:

`final case class ActionCoords(sid: Observation.Id, actCoords: ActionCoordsInSeq)`

`final case class ActionCoordsInSeq(stepId: StepId, execIdx: ExecutionIndex,
                                   actIdx: ActionIndex)`

Reply:
I moved it to `ActionCoordsInSeq`.